lib/src/single_line_comments/parser/single_line_comments.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:highlight/highlight_core.dart';
+import 'package:highlight/languages/dart.dart';
+import 'package:highlight/languages/go.dart';
+import 'package:highlight/languages/java.dart';
+import 'package:highlight/languages/python.dart';
+import 'package:highlight/languages/scala.dart';
+
+class SingleLineComments {
+  const SingleLineComments._();
+
+  static final byMode = <Mode, List<String>>{
```

Comment:
why not byLanguage?

Reply:
The users would have to guess if it is the `Mode` object or a string alias.